science/multimon-ng/files/patch-cmakelists.txt_x11.diff

```diff
@@ -0,0 +1,15 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 73c8422..1ca0af9 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -26,8 +26,8 @@ if( WIN32 )
+ elseif( UNIX )
+ 	find_package( X11 )
+ 	if ( X11_FOUND )
+-		option( X11_SUPPORT "Enable X11 display support" ${X11_FOUND} )
```

Comment:
If possible it would be better to enable/disable X11 via cnake configuration options, than a patch file.

Reply:
Unfortunately not, otherwise why I used the patch? And I haven't the time to send a patch upstream.